build.sbt

```diff
@@ -3,17 +3,18 @@ ThisBuild / organization := "me.shadaj"
 Global / onChangedBuildSource := ReloadOnSourceChanges
 turbo := true
 
-ThisBuild / libraryDependencies += compilerPlugin(scalafixSemanticdb)
 addCommandAlias("style", "compile:scalafix; test:scalafix; compile:scalafmt; test:scalafmt; scalafmtSbt")
 addCommandAlias(
   "styleCheck",
   "compile:scalafix --check; test:scalafix --check; compile:scalafmtCheck; test:scalafmtCheck; scalafmtSbtCheck"
 )
 
-val scala212 = "2.12.10"
-val scala213 = "2.13.2"
+val scala212 = "2.12.14"
+val scala213 = "2.13.6"
+val scala3   = "3.0.1"
 
-ThisBuild / scalaVersion := scala213
+ThisBuild / scalaVersion := scala3
```

Comment:
We'll need to keep the default version on `scala213` to get style checks to pass.